src/Agent/NewRelic/Agent/Core/Core.csproj

```diff
@@ -124,7 +125,7 @@
     </ItemGroup>
 
     <PropertyGroup>
-      <ILRepackIncludeCount Condition="'$(TargetFramework)' == 'net462'">14</ILRepackIncludeCount>
+      <ILRepackIncludeCount Condition="'$(TargetFramework)' == 'net462'">15</ILRepackIncludeCount>
       <ILRepackIncludeCount Condition="'$(TargetFramework)' == 'netstandard2.0'">11</ILRepackIncludeCount>
```

Comment:
It looks like we're only repacking that dependency on .net framework.

Reply:
It only needs to be packed for the net462 target - I also had to add a package reference for that target. (I thought I had it in the previous commit, but I didn't...)

Reply:
Are you sure about that for .net core? It's hard to know which dependencies are shipped with the different "frameworks" available in .net core. You might try testing it with a simple Console application that doesn't add any additional libraries except for maybe the NewRelic.Api nuget and then uses the transaction attribute to create a transaction. Simple console application seem to pull in the least amount of dependencies (from the framework) for .net core.

Reply:
I've been testing in a console app, targeting every tfm I can - net462, net471, coreapp31, net50, net60, net70 . Only need the package for net462.